The script webkit-filter-log is broken since the github migration

carlosgcampos opened the pull request:
#### f2bacb5cfc5f9996e51e2dda6e43b38e448afd7f
<pre>
The script webkit-filter-log is broken since the github migration
<a href="https://bugs.webkit.org/show_bug.cgi?id=242434">https://bugs.webkit.org/show_bug.cgi?id=242434</a>

Reviewed by Jonathan Bedard.

Just call git log with a simple format showing the info we need for the
directories containing changes specific to a given port.

* Tools/Scripts/webkit-filter-log:

Canonical link: <a href="https://commits.webkit.org/252265@main">https://commits.webkit.org/252265@main</a>
</pre>

JonWBedard reviewed (approved):
Would also like to point our `git-webkit log`, which will include identifiers and revisions (if available) along with revisions in the output.

carlosgcampos committed:
The script webkit-filter-log is broken since the github migration
https://bugs.webkit.org/show_bug.cgi?id=242434

Reviewed by Jonathan Bedard.

Just call git log with a simple format showing the info we need for the
directories containing changes specific to a given port.

* Tools/Scripts/webkit-filter-log:

Canonical link: https://commits.webkit.org/252265@main
